plugins/sourcegraph/access_token.go

```diff
@@ -0,0 +1,64 @@
+package sourcegraph
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func AccessToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.AccessToken,
+		DocsURL: sdk.URL("https://docs.sourcegraph.com/cli"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Username,
+				MarkdownDescription: "Your Sourcegraph username.",
+				Secret:              false,
+				Composition: &schema.ValueComposition{
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+						Symbols:   true,
```

Comment:
Quick note that Sourcegraph definitely supports symbols in the username:

<img width="913" alt="image" src="https://user-images.githubusercontent.com/18581859/212820812-46a342f2-2684-4c94-9c03-ef99f53287b8.png">